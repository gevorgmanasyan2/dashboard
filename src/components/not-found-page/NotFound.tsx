import React from 'react'
import { Result } from 'antd'
import './notFound.scss'
import { Link, useHistory } from 'react-router-dom'

const NotFound = () => {
  const history = useHistory()
  return (
    <div className='not__found__page'>
      <Result
        status="404"
        title="404"
        subTitle="Sorry, the page you visited does not exist."
        extra={
          <div className='not__found__page__content'>
            <p className='not__found__page__content__info'>
              Please &nbsp;<span onClick={history?.goBack}>return</span>&nbsp;to the previous page or go to&nbsp;
              <Link to="/">dashboard</Link>&nbsp;page
            </p>
          </div>
        }
      />
    </div>
  )
}

export default NotFound
