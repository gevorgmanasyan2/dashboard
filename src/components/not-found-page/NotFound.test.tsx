/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './NotFound'

it('should render NotFound component', () => {
  expect(shallow(<NotFound />)).toMatchSnapshot()
})
