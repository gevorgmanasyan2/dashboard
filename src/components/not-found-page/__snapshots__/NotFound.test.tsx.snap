// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound component 1`] = `
<div
  className="not__found__page"
>
  <Result
    extra={
      <div
        className="not__found__page__content"
      >
        <p
          className="not__found__page__content__info"
        >
          Please  
          <span>
            return
          </span>
           to the previous page or go to 
          <Link
            to="/"
          >
            dashboard
          </Link>
           page
        </p>
      </div>
    }
    status="404"
    subTitle="Sorry, the page you visited does not exist."
    title="404"
  />
</div>
`;
